src/main/java/org/junit/Assert.java

```diff
@@ -955,4 +955,96 @@ public static void assertEquals(Object[] expecteds, Object[] actuals) {
             Matcher<? super T> matcher) {
         MatcherAssert.assertThat(reason, actual, matcher);
     }
+
+    /**
+     * This interface facilitates the use of expectThrows from Java 8. It allows method references
+     * to void methods (that declare checked exceptions) to be passed directly into expectThrows
+     * without wrapping. It is not meant to be implemented directly.
+     *
+     * @since 4.13
+     */
+    public interface ThrowingRunnable {
```

Comment:
why the junit has still the same tendency to use nested interfaces and nested annotations like Parameter.
It's really awful code like Assert.ThrowingRunnable and using static import for Assert.* and much easier to use and import ThrowingRunnable. Bad example is Parameterized.Parameter.
The only reason why nested classes is that they are not static and share one instance; otherwise it's useless.